lib/screens/explore_screens/unplant_seeds/interactor/mappers/unplant_seeds_mapper.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/event_bus/event_bus.dart';
+import 'package:seeds/domain-shared/event_bus/events.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/explore_screens/unplant_seeds/interactor/viewmodels/unplant_seeds_page_commands.dart';
+import 'package:seeds/screens/explore_screens/unplant_seeds/interactor/viewmodels/unplant_seeds_state.dart';
+
+class UnplantSeedsResultMapper extends StateMapper {
+  UnplantSeedsState mapResultToState(UnplantSeedsState currentState, Result result) {
+    if (result.isError) {
+      print('Error transaction hash not retrieved');
+      return currentState.copyWith(
+        pageState: PageState.success,
```

Comment:
pageState: PageState.success -> pageState: PageState.Fail

Reply:
I dont want to how a full page error, just the snackbar if unplant fail